crates/parcel_config/src/pipeline.rs

```diff
@@ -0,0 +1,310 @@
+use std::path::Path;
+
+use glob_match::glob_match;
+use indexmap::IndexMap;
+
+use super::parcel_config::PluginNode;
+
+#[derive(Debug, Default, PartialEq)]
+pub struct PipelineMap {
+  map: IndexMap<String, Vec<PluginNode>>,
+}
+
+impl PipelineMap {
+  pub fn new(map: IndexMap<String, Vec<PluginNode>>) -> Self {
+    Self { map }
+  }
+
+  pub fn get(&self, path: &Path, pipeline: &Option<impl AsRef<str>>) -> Vec<PluginNode> {
```

Comment:
imo you could document what this structure does

Reply:
Will do